requirements/requirements-base.txt

```diff
@@ -1 +1,2 @@
+Django==1.10.6
```

Comment:
Can you remove Django since the CI will run the tests against [these](https://github.com/manosim/django-rest-framework-api-key/blob/master/tox.ini#L17-L19) versions of Django?